odoo/service/db.py

```diff
@@ -103,7 +103,7 @@ def _create_empty_database(name):
         if cr.fetchall():
             raise DatabaseExists("database %r already exists!" % (name,))
         else:
-            cr.autocommit(True)     # avoid transaction block
+            cr._cnx.autocommit = True   # avoid transaction block
```

Comment:
Thanks! :+1:

I was hoping we could remove the autocommit sections entirely, but that doesn't work because database-level operations fail when run inside a transaction block. Maybe the comments could do a better job of documenting it ;-)

```suggestion
            # database-altering operations cannot be executed inside a transaction
            cr._cnx.autocommit = True
```

Reply:
I will modify the comment accordingly, thanks a lot for the quick review!